qiskit_nature/mappers/second_quantization/qubit_mapper.py

```diff
@@ -107,7 +107,12 @@ def times_annihilation_op(position, pauli_table):
 
         # TODO to_list() is not an attribute of SecondQuantizedOp. Change the former to have this or
         #   change the signature above to take FermionicOp?
-        for label, coeff in second_q_op.to_list():
+        label_coeff_list = (
+            second_q_op.to_list(display_format="dense")
+            if isinstance(second_q_op, FermionicOp)
```

Comment:
I don't think we should have this isinstance check. Instead, we should propagate the `display_format` argument up to the `SecondQuantizedOp` interface.
It is fine that implementations other than `FermionicOp` ignore it for now.

Reply:
SpinOp and VibrationalOp do not have it. I considered adding the arguments to them, but I didn't think it's a good idea to add meaningless ones, so I left it like this for now.

Reply:
See below.